File: helloworld/src/main/java/com/codekul/helloworld/MainActivity.java
package com.codekul.helloworld;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onPlane(View view) {

        ImageView img = findViewById(R.id.imageView);
        img.setImageResource(R.drawable.html5);
    }

    public void onBus(View view) {

        ImageView img = findViewById(R.id.imageView);
        img.setImageResource(R.drawable.ic_logo);
    }
}

File: app/src/main/java/com/codekul/androidnsnov101130/MainActivity.kt
package com.codekul.androidnsnov101130

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val vw = View(this)
    }

    fun onBus(view: View?) {
        imageView.setImageResource(R.drawable.ic_logo)
    }

    fun onPlane(view: View?) {
        imageView.setImageResource(R.drawable.html5)
    }
}
